### DISCUSSION TITLE
`Menu` component: access uncontrolled `opened` state from `<Menu.Target>`

### DISCUSSION THREAD

**Discussion by mikhail-molchanov** [mantinedev/mantine#6965] [Feature requests]
Hi guys. Awesome work done with the library! 

Would it be an idea to let children of `<Menu.Target>` component access uncontrolled `opened` state vie e.g. render function?
Something like:
```js
<Menu.Target>
  ( {opened} ) => opened ? <IconChevronUp/> : <IconChevronDown>}
<Menu.Target/>
```

Could be helpful if target rendering depends on the `opened` state. At the moment, the only way I can think of is to use controlled menu, which feels a bit like an overkill for things like that.

**Comment by rtivital** [comment]
`Menu.Target` component sets `data-expanded="true"` attribute on the child element, it is enough to apply styles depending on the opened state. To get value in js, use controlled menu.

**Reply by mikhail-molchanov** [reply]
Okay, makes sense. Thanks for the feedback!
